.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: Build glossary
+on:
+  push:
+    branches:
+      - master
+  schedule:
+    - cron:  '0 */12 * * *'
+
+jobs:
+  run:
+    name: Build glossary
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v2
+
+    - name: Set up ruby
+      uses: ruby/setup-ruby@v1
```

Comment:
I think we want to pass in the Ruby version from `.ruby_version`. I'm sure something else does this already.

Reply:
The action either wants a version passed in from the `build.yml`, or will [auto-detect the `.ruby-version`](https://github.com/dxw/glossary/runs/753840677?check_suite_focus=true#step:3:6). It complains if you do neither.